contents/docs/csp-tracking/index.mdx

```diff
@@ -15,7 +15,14 @@ You can send reports of CSP rule violations to PostHog, which is useful for
 ## Where do I start?
 
 ### Create a new dashboard using the CSP template
-To get started, create a [new dashboard](https://app.posthog.com/dashboard#newDashboard=modal) and choose the "CSP Violation Reports" template.
+<ProductScreenshot
+    imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2025_05_19_at_22_43_16_55c2359415.png"
+    imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2025_05_19_at_22_42_53_f30b0d9662.png"
+    alt="CSP dashboard template"
+    classes="rounded"
+/>
+
+To get started, create a [new dashboard](https://app.posthog.com/dashboard?templateFilter=csp#newDashboard=modal) and choose the "CSP Violation Reports" template.
```

Comment:
Ahh I see what you mean with the filter now, this is awesome 😎 
```suggestion
To get started, create a [new dashboard](https://app.posthog.com/dashboard?templateFilter=csp#newDashboard=modal) and choose the **CSP Violation Reports** template.
```

Reply:
Yeah, we can probably scan through the others and use it there as well :D